packages/client/helpers/functional-test/run-tests.ts

```diff
@@ -157,6 +157,15 @@ async function main(): Promise<number | void> {
       }
     }
 
+    if (providerFlavors.includes(ProviderFlavors.VITESS_FK)) {
+      jestCli = jestCli.withArgs(['--runInBand'])
+      // jestCli = jestCli.withEnv({ TEST_REUSE_DATABASE: 'true' })
```

Comment:
Migrate has some trouble to drop tables that have foreign keys in some cases, so for now each database is independent.